aurora-mysql.cfhighlander.rb

```diff
@@ -35,5 +35,7 @@
     ComponentParam 'NamespaceId' if defined? service_discovery
 
     ComponentParam 'EnableReplicaAutoScaling', 'false', isGlobal: true
+
+    ComponentParam 'EnableCloudwatchLogsExports', defined?(log_exports) ? log_exports : '', isGlobal: true
```

Comment:
@petlitskiy The EnableCloudwatchLogsExports param does not need to be a global parameter can you please remove the isGlobal from thr parameter definition  can you also change for the EnableReplicaAutoScaling parameter as well

Reply:
@aaronwalker Done.